modules/nf-core/ganon/buildcustom/main.nf

```diff
@@ -0,0 +1,44 @@
+process GANON_BUILDCUSTOM {
+    tag "$meta.id"
+    label 'process_low'
```

Comment:
Might we need `process_medium` or `process_high` here?

Reply:
Fair point!